string eef_name
float64 angle
---
bool success
